xbmc/cores/VideoPlayer/DVDCodecs/Video/VDPAU.cpp

```diff
@@ -2450,6 +2486,9 @@ void CMixer::InitCycle()
 
   m_processPicture.DVDPic = m_mixerInput[1].DVDPic;
   m_processPicture.videoSurface = m_mixerInput[1].videoSurface;
+
+  // update deinterlacing method used in processInfo (none if progressive)
+  m_config.processInfo->SetVideoDeintMethod(deintStr);
```

Comment:
You prefer a member tracking it or can I base on m_Deint? e.g. at the beginning of this method before calling SetDeinterlacing if (method != m_Deint) -> update?